### PR TITLE
[release-4.13] OCPBUGS-18553: Disable rcu_expedited after node has finished booting

### PR DESCRIPTION
**Pull request by fontivan**
Cherry pick of OCPBUGS-17660 that was merged into the main branch 2 weeks ago
Due to issues with the CI I had to update the test to account for the new source CRs

### CODE REVIEW

**Comment by Missxiaoguo** [comment]
It's recommended to use "git cherry-pick -x" with the original commit of the test change. This preserves the author and commit title, making the commit history more consistent with the master branch. Additionally, you could cherry-pick the test change first, then the second commit can be eliminated.

File: ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go [hunk 1]
```diff
@@ -1071,6 +1071,7 @@ func Test_filterExtraManifests(t *testing.T) {
   exclude: %s 
   include: %s
 `
+	const testSourceCrPath = "./testdata/source-cr-extra-manifest-copy/"
```
Comment (imiller0) [inline ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go:1074]: I don't see any of the changes in this file in the original commit PR 1627. Can you please verify this backport is limited to the necessary changes.

Comment (fontivan) [reply ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go:1074]: Unfortunately in the 4.13 branch the CI tests here depend on the hard coded list of files in the source-crs path

After discussing with Nahian we found that this fix, to use separate testdata files, was made in the main branch some time ago and decoupled the test result from the file list in source crs

The CI will not pass without updating these tests in 4.13

**Review by imiller0 — APPROVED** [review]
/lgtm

**Pull request merged**